Menambahkan proyek awalan baru

ywnn opened the pull request:
Menambahkan starter project untuk Preact + Typescript + Parcel.

Commit:
Update README.md

Adding starter project Preact + Typescript.

Commit:
Update README-ID.md

Menambahkan proyek awalan untuk preact + typescript.

jefrydco reviewed (changes requested):
Hi, terima kasih telah menambahkan proyek ke repositori ini. Minta tolong dicek kembali ya format penulisan untuk starter projectnya, sepertinya kurang `kata kunci`

Commit:
Update README-ID.md

Memperbaiki kata kunci yang kurang pada proyek awalan serta memperbaiki tipe dari beberapa kontributor lainya.

Commit:
Update README.md

Fix missing keyword on starter project Preact + Typescript + ParcelJS

ghost commented:
> Hi, terima kasih telah menambahkan proyek ke repositori ini. Minta tolong dicek kembali ya format penulisan untuk starter projectnya, sepertinya kurang `kata kunci`

Baik, kak. Sedang diperbaiki, mohon koreksinya bila ada kekurangan.

jefrydco reviewed (approved):
Thank you!

jefrydco commented:
@ywnn Minta tolong resolve conflict nya ya

Commit:
Merge branch 'master' into master

ghost commented:
> @ywnn Minta tolong resolve conflict nya ya

Baik, kak. Sudah ditangani.
